test/test_common/environment.cc

```diff
@@ -49,6 +48,30 @@ void TestEnvironment::initializeOptions(int argc, char** argv) {
   argv_ = argv;
 }
 
+bool TestEnvironment::isTestIpVersionOnly(const Network::Address::IpVersion& type) {
+  const char* option = ::getenv("IP_TEST_TYPE");
```

Comment:
Will test/main.cc be overwritten by site specific versions of main.cc? https://github.com/lyft/envoy/blob/master/test/main.cc#L5